storybook/.storybook/main.ts

```diff
@@ -5,6 +5,7 @@ const config: StorybookConfig = {
   stories: [
     // TODO: remove this when every stories are migrated to `@osrd-project/storybook/stories/`
     '../../ui-*/src/stories/**/*.stories.@(js|jsx|mjs|ts|tsx)',
+    '../../ui-charts/src/**/stories/**/*.stories.@(js|jsx|mjs|ts|tsx)',
```

Comment:
This should already be covered by the glob pattern right above, no?

Reply:
Nope, the pattern above requires `stories` to be a direct child of `src`, that's the issue.

Reply:
Aha, makes sense.